0000-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,104 @@
+
+### Modeled Signal Strength 
+Date of CPI approval (only used as a tiebreaker if tie for attribute 1)
+
+Scoring attribute 2 (CPI approval date) will only be used when there is a tie for two or more radios for the proceeding attribute. 
+
+Please note, that the multiplier table above only affects the modeled coverage points that are given to each radio, and does not affect rewards distributed for the transfer of data. 
+
+To see how this HIP would affect a deployment of five (5) radios that provide coverage to the same res12 hex, see the below example:
+
+Radio A
+Signal Strength: -77.33 dBm
+CPI Approval Date: 05/01/2023 23:24:25
+Score: 1
+Coverage Points Awarded: 16 (16 * 1)
+
+Radio B 
+Signal Strength: -88.77dBm
+CPI Approval Date: 12/01/2022 01:01:01
+Score: 2
+Coverage Points Awarded: 12 (16 * .75)
+
+
+Radio C 
+Signal Strength: -88.77dBm
+CPI Approval Date: 12/02/2022 12:11:01
+Score: 3
+Coverage Points Awarded: 8 (16*.50)
+
+Radio D
+Signal Strength: 93.60 dBm
+CPI Approval Date: 12/05/2022 11:51:01
+Score: 4
+Coverage Points Awarded: 4 (16*.25) 
+
+Radio E 
+Signal Strength: 94.69 dBm
+CPI Approval Date: 08/01/2022 05:01:59
+Score: Fail
+Coverage Points Awarded: 0 (8*0)
+
+Since Radio A has the highest signal strength in that hex, it will be granted a 1X multiplier, which will award it with the full 16 (16 x 1) modeled coverage points for that epoch. 
+
+However, since radios, B, C, D and E have a lower signal strength than radio A, and are providing redundant coverage, they will not earn the full modeled coverage points. Instead, they will only earn a fraction of that amount depending on their score. 
+
+Since radios B and C tied in Signal Strength, the CPI approval date is used to determine which radio is scored next. For this example, radio B was CPI approved on 12/01/2022 01:01:01 and radio C was CPI approved on 12/02/2022 12:11:01. Therefore, radio B was CPI approved first, and will thus have a score of 2, while radio C having a score of 3. 
+
+Since radio E had the lowest signal strength out of all five (5) radios, and only the top four (4) radios will earn rewards, radio E will not earn any modeled coverage points.
+
+## Drawbacks:
+The implementation of this proposal could increase the complexity of the Mobile network, and modeled coverage scores. 
+
+## Rationale and Alternatives:
+An alternative would be to allow radios and hexes to keep earning the defined amount of modeled coverage points as described in HIP 74, which may prevent or stagnate the growth of the network. 
+
+## Unresolved Question:
+How will mappers and mapping of the network tie into this?
+
+## Deployment Impact:
+Outdoor radio deployers will now need to be cognizant of where they are placing their radios in order to maximize modeled coverage point. Additionally, there’s a large amount of overlapping coverage. Deployers may have to find new locations for some or all of their radios in order for them to continue to earn modeled coverage points. 
```

Comment:
Would be good to add here how helium core devs are expected to implement this. Is there code provided, more requirements, is the ambiguity of implementation left up to them? Just say something to help Developers read over this and where they are empowered to make this happen or anything they should consider during implementing.

Reply:
Is this question relating to the unresolved question section, or the deployment impact section? 

Also, is it referring to the overall implementation of this HIP, or a specific section?